.github/workflows/ci.yml

```diff
@@ -143,6 +143,7 @@ jobs:
     strategy:
       matrix:
         browser: [[chrome, chromium], [firefox, firefox], [edge, webkit]]
+      fail-fast: false
```

Comment:
Similar to the above `test` job, I thought it's better to run all matrix without `fail-fast` so that it's easier to identify potential browser-specific quirks easily.